### PR TITLE
discovar: fix a test block

### PR DESCRIPTION
**Pull request by tmozgach**
### Have you:

- [ ] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [ ] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [ ] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: discovar.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Discovar < Formula
   sha256 "c46e8f5727b3c8116d715c02e20a83e6261c762e8964d00709abfb322a501d4e"
   # doi "10.1038/ng.3121"
   # tag "bioinformatics"
+  revision 1
```
Comment (jonchang) [inline discovar.rb]: I don't think we need a new `revision` for just test block changes.

**Comment by tmozgach** [comment]
@sjackman could you please merge?

**Comment by sjackman** [comment]
Please remove the `revision 1` line, and I can merge it.

**Comment by tmozgach** [comment]
@sjackman done!

**Comment by sjackman** [comment]
Thanks, Tanya! And Jon for merging.

**Pull request merged**